docs/guides/registries/self-signed.md

```diff
@@ -68,7 +68,7 @@ global:
       http = true|false
       insecure = true|false
       ca=["<ca-path-pem>"]
-      [[registry."<registry-hostname>".keypair]]
+      [[registry."<registry-hostname>".key pair]]
```

Comment:
This shouldn't be changed as it introduces whitespace.

Reply:
Oh sorry, didn't notice it was a field.